src/model/object/entity/individual/individual.h

```diff
@@ -16,7 +16,7 @@ class Individual : public Entity {
     int getID();
     int getVisionRange();
 
-    Action * act();
+    virtual Action * act(std::vector<Object *> * visibles) = 0;
```

Comment:
why `= 0`?

Reply:
it's declaring this method abstract, since there is no default implementation which would make sense